doc/antora/modules/ROOT/pages/gethelp.adoc

```diff
@@ -0,0 +1,47 @@
+#Getting Help
+
+##Email and weblinks go here
+
+## Official Support 
+InkBridge Networks
+freeRADIUS org
+
+##How can I get training for my staff?
+Inkbridge Networks  offers a range of on-site training courses to meet your needs. Courses include:
+	• Introduction to RADIUS An introductory course for all administrators who are unfamiliar with
```

Comment:
Markdown should be just " *", and not multiple spaces and a bullet character.  Markdown will convert the "*" to a bullet when it's formatted for HTML, etc.